### DISCUSSION TITLE
RPBroadcastSampleHandler issue

### DISCUSSION THREAD

**Discussion by hardikthakkar2208** [HaishinKit/HaishinKit.swift#1364] [Q&A]
Hello Sir, 

I face one issue if I add the "Admob" framework in the application "RPBroadcastSampleHandler" does not give broadcasting data. Do you have any solution for this issue?

**Comment by shogo4405** [comment]
Please inquire about AdMob on Google's forums or similar platforms. Asking here may not lead to a solution.
